### ISSUE TITLE
Decal visibility behaves abnormally

### ISSUE THREAD

**Issue by carbonated-niko** [o3de/o3de#17617]
**Describe the bug**
I'm running into a handful of decal oddities.

1st - This is a default scene, with a 1x1 cube and a simple decal. I start play mode, and the decal vanishes. Stop play mode and its visible again.

https://github.com/o3de/o3de/assets/29128124/49996936-74e0-45b1-a810-d8adae9fae1f

2nd - I added another decal (white) into this scene. On starting playmode, this decal is now invisible. However, the original decal is now visible, but when play mode ends, the decal disappears and I have to toggle the objects visibility in the entity hierarchy.

https://github.com/o3de/o3de/assets/29128124/b4cabe07-5842-4aa3-87ab-4b423db25ef5

**Steps to reproduce**
1. Create a new level
2. Create an entity with a decal and a material
3. Start Play mode

Second issue

1. Create a new level
2. Create an entity with a decal and a material
3. Create another entity with the same setup
4. Start play mode

**Expected behavior**
Both decals are visible, and neither are invisible when play mode ends

**Actual behavior**
Visibility is inconsistently incorrect

**Found in Branch**
main

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Windows
 - Version: 10
 - CPU: Intel I9-10980XE
 - GPU: NVidia 3060Ti
 - Memory: 64 GB

**Comment by moudgils** [comment]
Yep This seems like a decal culling bug with the main pipeline. It can be picked up by anyone looking for bugs to fix.

**Comment by moudgils** [comment]
https://github.com/o3de/o3de/pull/17819
